src/shared/components/button (clients&team-members)/Button.module.css

```diff
@@ -0,0 +1,25 @@
+.buttonDefaultColor {
```

Comment:
Something is going wrong, the directory has the name of `button (clients&team-members)`.